assets/js/facebook-for-woocommerce-modal.js

```diff
@@ -47,4 +47,13 @@
 		$( '.wc-backbone-modal-content' ).removeClass( 'processing' ).unblock();
 	}
 
+
+	/**
+	 * Close the current modal.
```

Comment:
Good call wrapping that repeated line on this function!

```suggestion
	 * Closes the current modal.
```